common/test/acceptance/pages/lms/course_home.py

```diff
@@ -165,22 +153,23 @@ def go_to_section(self, section_title, subsection_title):
         Example:
             go_to_section("Week 1", "Lesson 1")
         """
-        section_index = self._section_title_to_index(section_title)
-        if section_index is None:
-            raise ValueError("Could not find section '{0}'".format(section_title))
+        subsection_webelements = self._get_subsections_as_selenium_webelements()
+        subsection_titles = [self._get_outline_element_title(sub_webel)
+                             for sub_webel in subsection_webelements]
 
         try:
-            subsection_index = self._subsection_titles(section_index + 1).index(subsection_title)
+            subsection_index = subsection_titles.index(unicode(subsection_title))
```

Comment:
It would be better to use `six.text_type` over `unicode` here (for python 3 compatibility).